tpcc/worker.go

```diff
@@ -148,7 +172,14 @@ func (w *worker) run(errCh chan<- error, wg *sync.WaitGroup) {
 				break
 			}
 		}
-		if _, err := t.run(w.db, rand.Intn(*warehouses)); err != nil {
+		if *noWait {
+			warehouseID = rand.Intn(*warehouses)
+		}
+
+		time.Sleep(time.Duration(t.keyingTime) * time.Second)
```

Comment:
shouldn't we skip the `time.Sleep` if we're in the `*noWait` case?

Reply:
Yes. Thanks for catching that. Done